jujugui/static/gui/src/app/components/deployment-flow/login/test-login.js

```diff
@@ -251,4 +253,122 @@ describe('DeploymentLogin', function() {
       </div>);
     expect(output).toEqualJSX(expected);
   });
+
+  it('can emit the login links when requested', function() {
```

Comment:
"emit"? Could be render/display. I usually associate emit with events, but could just be me.